sdk/network/arm-network-rest/samples-dev/expressRouteCircuitsListArpTableSample.ts

```diff
@@ -34,8 +34,7 @@ async function listArpTable(): Promise<void> {
       devicePath,
     )
     .post(options);
-  const poller = getLongRunningPoller(client, initialResponse);
-  const result = await poller.pollUntilDone();
+  const result = await getLongRunningPoller(client, initialResponse);
```

Comment:
The poller is still old HLC style and should be used as below:
```suggestion
  const poller = await getLongRunningPoller(client, initialResponse);
  const result = await poller.pollUntilDone();
```